laplace/baselaplace.py

```diff
@@ -1335,7 +1397,7 @@ def _nn_predictive_classification(
 
         vector_to_parameters(self.mean, self.params)
 
-        return py
+        return torch.as_tensor(py)
```

Comment:
Just to make sure: I suppose this is for typing?

Reply:
Yes!